js/src/main/scala/org/widok/bindings/HTML.scala

```diff
@@ -384,5 +385,42 @@ object HTML {
     case class Inline(contents: View*) extends Widget.Container[Inline] {
       val rendered = DOM.createElement("span", contents)
     }
+
+  }
+
+  case class Source() extends Widget[Source] {
+    val rendered = DOM.createElement("source").asInstanceOf[HTMLSourceElement]
+
+    def src(s: String) = { rendered.src = s; this }
+    def src(s: ReadChannel[String]) = { attribute("source", s); this }
```

Comment:
Oops, yes. Fixed and avoided redundant "element" variable and pushed new commit.